packages/nextjs-contentful-renderer/src/utils/replaceAll/replaceAll.ts

```diff
@@ -0,0 +1,2 @@
+export const replaceAll = (searchValue: string, replaceValue: string) => (str: string) =>
```

Comment:
If I have a list pattern in mdx content like: `{{year:list(2022|2023)}}`,  this function wont work as expected, as we need to escape`(` `)` `|`, so you might want to look at these changes that I added in the website repo: https://github.com/cmpsr/website/pull/16/files#diff-44fa34c8b3a952e815ae6eab5074b111f541fb3e6a410e6f8a9f985083e45c28

Reply:
added a unit test to handle this later https://github.com/cmpsr/composer/pull/429/files#diff-35390f2bda1600e0d3bf068ce9653e58c88721bd96987de16f5c9f756274d8c9R228